js/app.js

```diff
@@ -0,0 +1,62 @@
+$(document).ready(function() {
+
+    // STEP 1 : Populate the pantry 
+
+    var pantry = {
+        strong: ['glug of rum', 'slug of whisky', 'splash of gin'],
+        salty: ['olive on a stick', 'salt-dusted rim', 'rasher of bacon'],
+        bitter: ['shake of bitters', 'splash of tonic', 'twist of lemon peel'],
+        sweet: ['sugar cube', 'spoonful of honey', 'splash of cola'],
+        fruity: ['slice of orange', 'dash of cassis', 'cherry on top'],
+    };
+
+    //step 2 : make constructor to fetch yes/no values
+
+    function prefCreator(strong, salty, bitter, sweet, fruity) {
```

Comment:
something like this you mean?
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments